assets/app/scripts/controllers/build.js

```diff
@@ -120,10 +129,16 @@ angular.module('openshiftConsole')
               delete $scope.buildConfigBuildsInProgress[buildConfigName][buildName];
             }
           }
-
-          $scope.canBuild = !hashSize($scope.buildConfigBuildsInProgress[buildConfigName]);
         }));
 
+        $scope.isPaused = function() {
+          return BuildsService.isPaused($scope.buildConfig);
+        };
+
+        $scope.canBuild = function() {
+          return BuildsService.canBuild($scope.buildConfig, $scope.buildConfigBuildsInProgress);
+        };
```

Comment:
Same comment, this should probably just be a boolean set in the scope from the watch rather than a function called from the view.